presto-orc/src/main/java/com/facebook/presto/orc/metadata/CompressionKind.java

```diff
@@ -15,5 +15,5 @@
 
 public enum CompressionKind
 {
-    UNCOMPRESSED, ZLIB, SNAPPY
+    UNCOMPRESSED, ZLIB, SNAPPY, ENCRYPTED
```

Comment:
> @martint, Presto defined its own CompressionKind. Do we have to have exact match between this one and the Apache one? For example, 'UNCOMPRESSED' is only roughly mapped to 'NONE' there. Thanks!

Ours is a subset. The names don't mean anything -- what matters is the number associated with each enum entry:

```
enum CompressionKind {
 NONE = 0;
 ZLIB = 1;
 SNAPPY = 2;
 LZO = 3;
 LZ4 = 4;
 ZSTD = 5;
}
```

Once we add support for LZO/LZ4/ZSTD we'll add the corresponding constants